### PR TITLE
Filtering by tags with AND and/or  OR condition has been added

### PR DESCRIPTION
**Pull request by kkateq**

### CODE REVIEW

File: www/script/project/rtBuilders.js [hunk 1]
```diff
@@ -38,12 +41,44 @@ define(function (require) {
             realtimePages.initRealtime(realtimeFunctions);
 
             $searchField = $(".dataTables_filter>label input");
+            var addTagsFilter = $("#btn-and-tag-select");
```
Comment (opdude) [inline www/script/project/rtBuilders.js]: did you mean andTagsFilter?

Comment (kkateq) [inline www/script/project/rtBuilders.js]: yes, add 'and' tags filter I meant :)

**Comment by kkateq** [comment]
Variable has been renamed. Please review changes.

**Comment by opdude** [comment]
We should test the functionality closely on this one :), but go ahead and merge

**Pull request merged**